.github/workflows/test_ruby.yml

```diff
@@ -29,13 +29,14 @@ jobs:
       matrix:
         include:
           # Test both FFI and Native implementations on the highest and lowest
-          # Ruby versions for CRuby and JRuby, but only on Bazel 5.x.
+          # Ruby versions for CRuby and JRuby
           - { name: Ruby 3.0, ruby: ruby-3.0.2, ffi: NATIVE }
           - { name: Ruby 3.0, ruby: ruby-3.0.2, ffi: FFI, continuous-only: true }
           - { name: Ruby 3.1, ruby: ruby-3.1.0, continuous-only: true }
           - { name: Ruby 3.2, ruby: ruby-3.2.0, continuous-only: true }
-          - { name: Ruby 3.3, ruby: ruby-3.3.0, ffi: NATIVE }
-          - { name: Ruby 3.3, ruby: ruby-3.3.0, ffi: FFI }
+          - { name: Ruby 3.3, ruby: ruby-3.3.0, continuous-only: true }
+          - { name: Ruby 3.4, ruby: ruby-3.4.1, ffi: NATIVE, image: 'us-docker.pkg.dev/protobuf-build/containers/test/linux/ruby:7.1.2-ruby-3.4.1-90d207f4e749b54c8792bbe974dfc70323b6566e' }
```

Comment:
Why override the image here?  this should be deducible from the ruby version, we just need to make sure all the versions got built at that commit

Reply:
It was expedient during development to just worry about 3.4 without worrying about changes to anything else. I'll check to see if making it uniform buys any trouble.

Reply:
Removed the override for 3.4, but added one for 3.0 because a ruby-gems update (that now requires Ruby 3.1) is blocking rebuilding the 3.0.2 docker image from the latest base image.